pages/exhibition/_exhibition/credits.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <div
+    data-qa="exhibition credits page"
+  >
+    <b-container class="pb-3">
+      <b-row>
+        <b-col
+          cols="9"
+          class="pb-3"
+        >
+          <h1>{{ $t('exhibitions.credits') }}</h1>
+          <!-- eslint-disable vue/no-v-html -->
+          <div
+            data-qa="credits text"
+            v-html="credits"
+          />
+          <!-- eslint-enable vue/no-v-html -->
+        </b-col>
+      </b-row>
+      <b-row v-if="page.hasPart">
+        <b-col>
+          <ExhibitionChapters
+            :exhibition-identifier="page.identifier"
+            :chapters="page.hasPart"
+            :credits="page.credits"
+          />
+        </b-col>
+      </b-row>
+    </b-container>
+  </div>
+</template>
+
+<script>
+  import marked from 'marked';
+  import createClient from '../../../plugins/contentful';
+  import ExhibitionChapters from '../../../components/exhibition/ExhibitionChapters';
+
+  export default {
+    components: {
+      ExhibitionChapters
+    },
+    computed: {
```

Comment:
add a head section with 
`title: this.page.title`
Perhaps interpolate a translation for '[title] - credits' and use 'exhibitions.credits' from the the locale file.